charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -71,9 +90,37 @@ subjects:
   apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: ClusterRole
-  name:  pp-plugins-scheduler
+  name:  system:pp-plugins-scheduler
```

Comment:
do we need `system` here?

Reply:
same, I followed the ClusterRole in the metrics-server